x-pack/test/performance/journeys/security_detections_dashboard/security_detections_dashboard.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { StepCtx } from '../../services/performance';
+
+export default function ({ getService }: FtrProviderContext) {
+  describe('security_users_dashboard', () => {
+    const performance = getService('performance');
+    const esArchiver = getService('esArchiver');
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/performance/es_archives/security_dashboard_data');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/performance/es_archives/security_dashboard_data');
+    });
```

Comment:
@achuguy you need to move it to the config file:

```
    testData: {
      esArchives: ['x-pack/test/performance/es_archives/security_dashboard_data'],
    },
```
You can use promotion_tracking_dashboard as example

Reply:
it will referenced in the output file and clarifies what data was used